### PR TITLE
Slh army

### PR DESCRIPTION
**Pull request by MarieEtienne**
I open the PR for your group

### CODE REVIEW

**Comment by S3B2ZO** [comment]
Thank you, we were going to make another one after following your recommendations.

**Comment by S3B2ZO** [comment]
Can you see if our pull request is ok for you ?

**Review by KlerviChoulette — APPROVED** [review]

**Review by Alexia-Gueguen — COMMENTED** [review]
I would like to give a review but unfortunately I can't see your project. They say the files were deleted.

**Comment by S3B2ZO** [comment]
Alexia I think you can show our file in "fichier_r.qmd" it is not deleted

**Comment by BastienBerrand** [comment]
@S3B2ZO Your code is running well and is well structured, I think you can merge.

**Review by BastienBerrand — APPROVED** [review]
Your code is running well and is well structured, I think you can merge.

File: fichier_r.qmd [hunk 1]
```diff
@@ -1,178 +1,169 @@
...
 
-#### Data processing
+## B. Processing of the abiotic dataset
 
 -   Firstly we concatenate all the abiotic data together
 
--   Then we start to clean the abiotic the dataset by deleting all years that precede the first event of capture, the we use the Lubridate R package [@grolemund2016lubridate] to formate the data column as we want and the dplyr R package [@wickham2014dplyr] to re-organize the data set
+-   Then we start to clean the abiotic the dataset by deleting all years that precede the first event of capture, the we use the Lubridate R package (@spinu_lubridate_2010) to formate the data column as we want and the dplyr R package (@wickham_dplyr_2023) to reorganize the data set.
 
 ```{r}
```
Comment (MarineMS) [inline fichier_r.qmd:49]: Maybe you could add titles to your chunks to make them more precise and avoid losing your readers.

File: fichier_r.qmd [hunk 2]
```diff
@@ -1,178 +1,169 @@
...
-
 ```
 
-#### Data processing
+## B. Processing of the abiotic dataset
 
 -   Firstly we concatenate all the abiotic data together
 
--   Then we start to clean the abiotic the dataset by deleting all years that precede the first event of capture, the we use the Lubridate R package [@grolemund2016lubridate] to formate the data column as we want and the dplyr R package [@wickham2014dplyr] to re-organize the data set
+-   Then we start to clean the abiotic the dataset by deleting all years that precede the first event of capture, the we use the Lubridate R package (@spinu_lubridate_2010) to formate the data column as we want and the dplyr R package (@wickham_dplyr_2023) to reorganize the data set.
```
Comment (manon-verdier) [inline fichier_r.qmd:47]: I think you wanted to say "clean the antibiotic dataset" here?

**Review by MarineMS — COMMENTED** [review]
For me your code is fine, I just suggested putting titles to your chunks to make it clear what you're doing in each part of the code.

File: fichier_r.qmd [hunk 3]
```diff
@@ -1,178 +1,169 @@
...
 points(data_abiot$LON,data_abiot$LAT,pch=20,col='green')
 points(coord_chiro$LON,coord_chiro$LAT,pch=20,col="blue")
-
 ```
 
-Data import :
-
-```{r}
-data_merged = read.csv("data_merged.csv", header = T)
-```
```
Comment (manon-verdier) [inline fichier_r.qmd:155]: the data we obtain showS

File: fichier_r.qmd [hunk 4]
```diff
@@ -1,178 +1,169 @@
...
+  inner_join(coord_chiro, by = "LIEU_DIT") 
 
 data_chiro <- data_chiro%>%
-  select(COMMUNE.x,LIEU_DIT,Date,Espèce,Sexe,Repro,Age,Year,julian_day,LAT,LON)%>% #on spécifie les colonnes que l'on garde
+  select(COMMUNE.x,LIEU_DIT,Date,Espèce,Sexe,Repro,Age,Year,julian_day,LAT,LON)%>% 
   rename(COMMUNE = COMMUNE.x)
 ```
 
-représentation graphique de la guyane avec les station météorologique et les points de capture
+## **D. Graphical representation of French Guiana with the incorporation of meteorological stations and capture points**
```
Comment (manon-verdier) [inline fichier_r.qmd:139]: You should clarify which one is in blue and which one is in green

File: fichier_r.qmd [hunk 5]
```diff
@@ -264,23 +252,516 @@ for(d in dd){
...
+
+ggplot(data_final, aes(Température, repro_positive)) +
+  geom_point() + 
+  geom_smooth(method = "loess") +
+  scale_y_continuous(trans='log10')
+```
+
+At this point, we might be tempted to see whether or not these climatic parameters influence the proportion of reproductive individuals... let's try!!
+
+## C. Study of climatic influence on the proportion of breeding indidivids
```
Comment (manon-verdier) [inline fichier_r.qmd:386]: individuals

**Review by manon-verdier — COMMENTED** [review]
I correct some minor typo, otherwise your analysis seems nice. I am however not a specialist of bayesians methods.

**Review by manon-verdier — APPROVED** [review]

**Pull request merged**